src/client/theme-default/styles/vars.css

```diff
@@ -358,3 +358,14 @@
   --vp-home-hero-image-background-image: none;
   --vp-home-hero-image-filter: none;
 }
+
+/**
+ * Component: Badge
+ * -------------------------------------------------------------------------- */
+
+:root {
+  --vp-c-badge-type-warning: #e7c000;
+  --vp-c-badge-type-tip: #42b983;
+  --vp-c-badge-type-error: #da5961;
+  --vp-c-badge-type-info: #0170fe;
```

Comment:
What about reusing current color palette?

Reply:
@meteorlxy 

Do you mean the colors defined in the `:root`block?
I think that's a good idea since it will handle the light and dark theme.
But I've to find similar colors there.

Reply:
I've just commited a change using the existing variables. I left the `c-badge` variables for more flexibility. For me it looks fine in light and dark theme now.